apps/system/lockscreen/style/notifications.css

```diff
@@ -354,26 +354,33 @@
 }
 
 /* Right-To-Left layout */
-html[dir="rtl"] #lockscreen-alt-camera {
-  float: left;
+:-moz-dir(rtl) #notifications-lockscreen-container .notification > img {
+  float: right;
 }
 
-html[dir="rtl"] #notifications-lockscreen-container .notification > div {
-  margin-left: 0;
-  margin-right: 5rem;
-  text-align: right;
+:-moz-dir(rtl) #notifications-lockscreen-container .notification::before {
+  left: unset;
+  right: 0;
 }
 
-html[dir="rtl"] #notifications-lockscreen-container .notification > div.title-container .title {
-  margin-right: 0;
-  margin-left: .5rem;
+:-moz-dir(rtl) #notifications-lockscreen-container .notification::after {
+  left: unset;
+  right: 0;
 }
 
-html[dir="rtl"] #notifications-lockscreen-container .notification > div.title-container .timestamp {
+:-moz-dir(rtl) #notifications-lockscreen-container .button-actionable {
+  border-left: unset;
+  border-right: 0.1rem rgba(255, 255, 255, 0.15) solid;
+  left: unset;
+  right: calc(100% - var(--width-actionable-button) + 1.4rem);
+}
+
+:-moz-dir(rtl) #notifications-lockscreen-container .notification > .title-container .timestamp {
   left: 1.5rem;
-  right: auto;
+  right: unset,
```

Comment:
This is causing a lint error. I'm not sure if this rule is necessary for the problem reported in the bug, since the bug is fixed even with this rule being ignored. But we should fix this anyway.